### DISCUSSION TITLE
【求助】光映广场有没有什么餐厅推荐？

### DISCUSSION THREAD

**Discussion by yuleiiiiii** [share121/inter-knot#856] [其他]
最近和朋友想去光映广场附近玩，想问问那边有没有什么好吃的或者特色餐厅之类的，最好是每周四或别的日子会有活动优惠的那种

**Comment by nailbiting21** [comment]
必定是麦肯基，每周有一天有活动

**Comment by fairy-404** [comment]
去麦当当

**Comment by zhangshshhhh** [comment]
我不知道光映广场有什么，但我知道仙舟罗浮上有一家官方餐厅

**Comment by dongzhiaidao** [comment]
那边有家会员制餐厅，玩累了可以直接睡觉，没事的

**Reply by Y-0930** [reply]
睡醒了屁股有点痛是为什么

**Comment by sky27950** [comment]
光映广场的火锅店很好吃，只不过真的很辣，建议点微辣的

**Comment by BITE-7274** [comment]
火锅店有什么好的，店门口才有好的
![20240714210847](https://github.com/user-attachments/assets/aaf75235-4326-4ce4-8ad9-e4d5a5730af7)

**Comment by ZhuYuan-148** [comment]
光辉广场的那家火锅店还不错，就是有点辣。

**Comment by yuleiiiiii** [comment]
感谢各位的推荐，我和朋友打算去把推荐的每一家店都尝试一下[]~(￣▽￣)~*，以及去会员制餐厅和火锅店之前需要预定肛肠科医院吗(っ °Д °;)っ
